daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/ScenarioRunner.scala

```diff
@@ -53,6 +60,15 @@ final case class ScenarioRunner(
     }
   }
 
+  private[this] def nextSeed(submissionSeed: crypto.Hash): crypto.Hash =
+    crypto.Hash.deriveTransactionSeed(
```

Comment:
I don’t particularly like any of the seed handling here and I’m almost certain the current behavior is not what anyone intended but for now this at least keeps the behavior.